src/cli/admin/admin.ts

```diff
@@ -27,6 +27,8 @@ export default function setup() {
     'List oauth2 clients with admin privileges.'
   );
 
+  program.command('export', 'Export everything.');
```

Comment:
Should we change the command name and help message to be a little more specific? I could see some confusion about what the "export" command does without seeing the help message, and confusion about what "export everything" entails.

Reply:
Yes, a more descriptive name and description would definitely be better here. I'm not sure what to call it though since I'm not sure what word describes everything that is being exported except maybe "config" because it is just exporting everything. Maybe the name could be "export-full-cloud-config" and the description could be "Export full cloud configuration for all ops that currently support export."